BLSQ/iaso
# IA-2752: GPS points for dummy data should be placed in Burkina Faso (not in the US)

Related JIRA tickets : [IA-2752](https://bluesquare.atlassian.net/browse/IA-2752)

## Self proofreading checklist


## Changes

Replaced the empty submission GPS with the org unit GPS point!

## How to test

From setuper folder, launch setuper via `python3 setuper.py` to generate test data and open the map tab on submissions page : 
[http://localhost:8081/dashboard/forms/submissions/tab/map/](http://localhost:8081/dashboard/forms/submissions/tab/map/)

You should see the GPS points placed in **Burkina Faso**!

## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/35a314e6-ccac-4c5d-bcc0-bf0786b57c20





[IA-2752]: https://bluesquare.atlassian.net/browse/IA-2752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
